src/data.seed.ts

```diff
@@ -16,6 +16,7 @@ export interface StationInterface {
 
 export class SeedDetails {
     organizationId: string | undefined;
```

Comment:
remove this property , we can use Organization property defined below everywhere

Reply:
fixed

Reply:
removing this property affected Pathways. Changed the values accordingly.